src/app/shared/lib/marked.ts

```diff
@@ -0,0 +1,23 @@
+import { MarkedOptions, MarkedRenderer } from "ngx-markdown";
+
+export function markedOptionsFactory(): MarkedOptions {
+  const renderer = new MarkedRenderer();
+  const linkRenderer = renderer.link;
+
+
+  renderer.link = (href, title, text) => {
+    const html = linkRenderer.call(renderer, href, title, text);
+    return html.replace(/^<a /, '<a target="_blank" ');
```

Comment:
I think we should add a test that verifies that the function here returns the expected HTML, when `href` is an invalid value. This test will also help explain the purpose of overriding this renderer (useful for future devs).

What do you think @seanlowjk  ?

Reply:
Yes, this would definitely be good! 

Apologies for the late reply ><